srcpkgs/qutebrowser/template

```diff
@@ -22,9 +22,7 @@ desc_option_webengine="Build Qt5 WebEngine support"
 
 if [ "$XBPS_TARGET_ENDIAN" = "le" ]; then
 	# qt5-webengine is only available for little-endian systems
-	if [ -z "$CROSS_BUILD" ]; then
-		# qt5-webengine can only be built when target and host bitness match
-		# python3-PyQt5-webengine is currently nocross
+	if [ "$XBPS_WORDSIZE" = "$XBPS_TARGET_WORDSIZE" ]; then
```

Comment:
Shouldn't this just copy the switch for `python3-PyQt5-webengine`? This will be broken for BE ppc when it should just pull `python3-PyQt5-webkit`.

Reply:
Nah, `python3-PyQt5-webengine` doesn't even need a `broken=anything`, since `qt5-webengine` is already marked as broken and a dep of it.

Reply:
This is different, because the package shouldn't fail to build if webengine can't be built; it should just fall back to webkit.

Reply:
Reading this again, I agree. The word-size test seems best.

Reply:
I guess this is settled to current form, then?

Reply:
I think that's reasonable